fix: remove mentions about github discussions

leeederek opened the pull request:
- remove mentions about github discussions
- point folks to new github issue templates
- add our new email link
- encourage engagement via discord & calendly 1:1 sessions

Changelog picked up from commits here:

fix: remove mentions about github discussions

mieubrisse commented on commons/networks/service_network_config_test.go:
```diff
@@ -76,6 +76,33 @@ func TestDisallowingNonexistentDependencies(t *testing.T) {
 
 // TODO test configuration IDs get incremented!
 
+func TestNetworkIdsDifferent(t *testing.T) {
```
Thsi should be `TestConfigurationIdsDifferent`

galenmarchetti replied:
done

mieubrisse commented on commons/networks/service_network_config_test.go:
```diff
@@ -76,6 +76,33 @@ func TestDisallowingNonexistentDependencies(t *testing.T) {
 
+func TestNetworkIdsDifferent(t *testing.T) {
+	testImage := "testImage"
+	expectedOrder := []int{
```
So this actually makes an assertion about the IDs that will come back (i.e. that they'll be sequentially-increasing ints starting at 0), which implicitly means this test knows about how the internals of the function work. IMO that should be abstracted and we should instead just assert that they're different - make a set-of-IDs-returned and assert that several calls to both `AddStaticImageConfiguration` and `AddTestImageConfiguration` don't collide with anything that already exists

galenmarchetti replied:
So the goal of this test is: "Add test to ensure that configuration IDs are incremented in the call to SvcNetworkCfgBuilder.AddConfiguration". Do we not want to test that they're incremented, but rather just that they're different?

galenmarchetti replied:
and changed regardless

leeederek committed:
fix: remove mentions about github discussions

- remove mentions about github discussions
- point folks to new github issue templates
- add our new email link
- encourage engagement via discord & calendly 1:1 sessions

victorcolombo commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+If you notice an issue in our docs (e.g. typo, grammar, conflicting information), search if there is already an issue open for it. Otherwise, please file a [docs issue report](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml&title=%5BDocs%5D%3A+) and we will address it as soon as we can.
+
+
+### Questions, help, or feedback
```
I wonder if makes sense to move this to README for greater visibility?

leeederek replied:
I am in support of moving this to the main README. In fact, the main README was where I first went to look for mentions about "github discussions".

Are you suggesting we move all of this to the main README or just this docs bit?

victorcolombo replied:
I was thinking about the `Questions, help or feedback` that is the most important section for users, and we should hide away anything related to contribution that will only affect <1% of users

leeederek replied:
Awesome, I agree. I've opened this PR for that: https://github.com/kurtosis-tech/kurtosis/pull/111

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+### Feature Requests
 
-If you have a feature request, search if a thread already exists in [Github Discussions](https://github.com/kurtosis-tech/kurtosis/discussions/categories/ideas). If a related discussion doesn't exist, you can open a [new discussion](https://github.com/kurtosis-tech/kurtosis/discussions/new?category=ideas). 
+If you have a feature request, search if a [feature requests already exists](https://github.com/kurtosis-tech/kurtosis/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22). If a related feature request doesn't exist, you can open a [feature request](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&template=feature-request.yml&title=%5BFR%5D%3A+). 
```
```suggestion
If you have a feature request, search if a similar [feature request already exists](https://github.com/kurtosis-tech/kurtosis/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22). If a related feature request doesn't exist, you can open a [feature request](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&template=feature-request.yml&title=%5BFR%5D%3A+). 
```

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+### Questions, help, or feedback
+
+If you have a question around how Kurtosis works and the [docs](https://docs.kurtosis.com) aren't enough, we're more than happy to help via the following ways:
+- Chat to us in our [Discord server](https://discord.gg/Es7QHbY4)
```
```suggestion
- Get help in our [Discord server](https://discord.gg/Es7QHbY4)
```

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+If you have a question around how Kurtosis works and the [docs](https://docs.kurtosis.com) aren't enough, we're more than happy to help via the following ways:
+- Chat to us in our [Discord server](https://discord.gg/Es7QHbY4)
+- Email us at [feedback@kurtosistech.com](mailto:feedback@kurtosistech.com)
+- Get dedicated help in a [1:1 session with us](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```
```suggestion
- Schedule dedicated help in a [1:1 session with us](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+### Documentation
 
-If you have a question around how Kurtosis works, search if you can find the answer in the [docs](https://docs.kurtosis.com). If you can't find your answer in the docs, ask a question [here](https://github.com/kurtosis-tech/kurtosis/discussions/categories/q-a). 
+If you notice an issue in our docs (e.g. typo, grammar, conflicting information), search if there is already an issue open for it. Otherwise, please file a [docs issue report](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml&title=%5BDocs%5D%3A+) and we will address it as soon as we can.
```
```suggestion
If you notice an issue in our docs (e.g. typo, grammar, conflicting information), first search to see if there is [already an issue open for it](https://github.com/kurtosis-tech/kurtosis/issues?q=is%3Aopen+is%3Aissue+label%3Adocs). Otherwise, please file a [docs issue report](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=leeederek&labels=docs&template=docs-issue.yml&title=%5BDocs%5D%3A+) and we will address it as soon as we can.
```

leeederek committed:
Update CONTRIBUTING.md

leeederek committed:
Update CONTRIBUTING.md

leeederek committed:
Update CONTRIBUTING.md

leeederek committed:
Update CONTRIBUTING.md

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+If you have a question around how Kurtosis works and the [docs](https://docs.kurtosis.com) aren't enough, we're more than happy to help via the following ways:
+- Get help in our [Discord server](https://discord.gg/Es7QHbY4)
+- Email us at [feedback@kurtosistech.com](mailto:feedback@kurtosistech.com)
+- Schedule dedicated help in a [1:1 session with us](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```
```suggestion
- Schedule a 1:1 session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```

leeederek commented on CONTRIBUTING.md:
```diff
@@ -2,17 +2,25 @@
+
+### Questions, help, or feedback
+
+If you have a question around how Kurtosis works and the [docs](https://docs.kurtosis.com) aren't enough, we're more than happy to help via the following ways:
```
```suggestion
If you have a question around how Kurtosis works and the [docs](https://docs.kurtosis.com) aren't enough, we're more than happy to help and chat about your use case via the following ways:
```

leeederek committed:
Update CONTRIBUTING.md

leeederek committed:
Update CONTRIBUTING.md

leeederek committed:
Merge branch 'main' into derek/update-readme-with-new-guidance

leeederek committed:
Merge branch 'main' into derek/update-readme-with-new-guidance
